app/components/message_attachments/action_button/action_button.js

```diff
@@ -28,19 +30,27 @@ export default class ActionButton extends PureComponent {
     }, 4000);
 
     render() {
-        const {name, theme, disabled} = this.props;
+        const {name, theme, disabled, style: customStyle} = this.props;
```

Comment:
If `style` is renamed, this simplifies and conflict with theme `style` is automatically handled.